src/lib.rs

```diff
@@ -21,12 +21,58 @@ use std::fmt::Write;
 use tree_sitter::{Language, Node};
 
 pub struct FormatOptions {
+    /// Number of spaces used for one level of indentation
     pub indentation: usize,
+    /// Wether to align prefix IRIs,
+    /// or use a single space after the ':'.
+    pub align_prefix_iris: bool,
+    /// Wether to re-indent comments,
+    /// ensuring they all have exactly one space after the '#'.
+    pub unify_comment_indents: bool,
+    /// Wether to sort subjects
+    pub sort_subjects: bool,
+    /// Wether to sort predicates within a subject
+    pub sort_predicates: bool,
+    /// Wether to sort objects withing one subject-predicate pair
+    pub sort_objects: bool,
```

Comment:
What about merging these options into a single `sort_terms` option?

Reply:
Also fine for me.
Though, you already sort prefixes (always). would we keep that, and just make it clear in the doc comments what gets always sorted and what only with this option enabled?

Reply:
Yes! Hence the "sort_terms" naming.